build.assets/flake/flake.nix

```diff
@@ -54,16 +47,13 @@
           nodeProtocTsVersion = "v5.0.1";
           grpcToolsVersion = "1.12.4";
           libpcscliteVersion = "1.9.9-teleport";
-          rustVersion = "1.68.0";
-          yarnVersion = "1.22.19";
 
-          overlays = [ (import rust-overlay) ];
+          overlays = [];
```

Comment:
I couldn't figure out a way to remove this line and still have it work, so I kept it empty instead.  😅